.github/workflows/publish.yml

```
@@ -2,7 +2,8 @@ name: Publish
 on: [push]
 concurrency: ci-${{ github.ref }}
 jobs:
-  build:
+  publish:
+    if: ${{ github.repository_owner == 'FabricMC' }}
```

I dont think you need the `${{`

```suggestion
    if: github.repository_owner == 'FabricMC'
```

Should do it as far as I can tell, im not 100% sure what you have here will work.

---

this syntax is from https://github.com/FabricMC/yarn/blob/544daf01e8e119b20b527b0a8a6780ceb2746575/.github/workflows/build.yml#L17 and it works for my projects in my organization

---

Ok sure, blame me then 👍 lets go with this.